src/workerd/api/streams/standard.h

```diff
@@ -100,8 +101,12 @@ struct UnderlyingSink {
   JSG_STRUCT(type, start, write, abort, close);
   JSG_STRUCT_TS_OVERRIDE(<W = any> {
     write?: (chunk: W, controller: WritableStreamDefaultController) => void | Promise<void>;
+    start?: (controller: WritableStreamDefaultController) => void | Promise<void>;
+    abort?: (reason: any) => void | Promise<void>;
+    close?: () => void | Promise<void>;
```

Comment:
Just commenting to braindump -- the reason for these overrides is that these class members aren't declared as functions -- they're declared as variables of a function type. This isn't currently picked up by our extraction script, so we've just defined a few overrides here and in `trace.h` to cover these. `handleEvent` is similar, except it's also got some inheritance stuff going on? @penalosa can clarify